.github/workflows/release.yml

```
@@ -0,0 +1,110 @@
+  contents: write
+
+jobs:
+  release-gate:
+    runs-on: macos-26
+    timeout-minutes: 240
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Select Xcode
+        run: |
```

actionlint (1.7.11)

[error] shellcheck reported issue in this script: SC2010:warning:9:3: Don't use ls | grep. Use a glob or a for loop with a condition to allow non-alphanumeric filenames

(shellcheck)